sentry/src/test/java/io/sentry/HubTest.kt

```diff
@@ -16,6 +17,8 @@ import io.sentry.hints.SessionEndHint
 import io.sentry.hints.SessionStartHint
 import io.sentry.protocol.SentryId
 import io.sentry.protocol.User
+import org.mockito.ArgumentCaptor
+import org.mockito.Mockito.`when`
```

Comment:
H: same for here, https://github.com/nhaarman/mockito-kotlin/wiki/Mocking-and-verifying#stubbing
`whenever`